src/dhcp_proxy/lib.rs

```diff
@@ -174,27 +174,50 @@ impl NetworkConfig {
     ///
     /// * `p`: path to uds
     ///
-    /// returns: Result<NetavarkProxyClient<Channel>, Status>
+    /// returns: Result<NetavarkProxyClient<Channel>, NetavarkError>
     ///
     /// # Examples
     ///
     /// ```
     ///
     /// ```
-    async fn get_client(p: String) -> Result<NetavarkProxyClient<Channel>, Status> {
+    async fn get_client(p: String) -> Result<NetavarkProxyClient<Channel>, NetavarkError> {
         // We do not know why the uds connections need to be done like this.  The
         // maintainer suggested it is part of the their API.
-        let endpoint = Endpoint::try_from("http://[::1]:10000")
-            .map_err(|e| Status::internal(e.to_string()))?;
+        // We know this is safe and if it ever fails test will catch it
+        let endpoint = Endpoint::try_from("http://[::1]").unwrap();
```

Comment:
This is missing the port from the current version, is that intended?

Reply:
yes